src/host/ut_host/TextBufferTests.cpp

```diff
@@ -2014,6 +1991,62 @@ void TextBufferTests::TestBurrito()
     VERIFY_IS_FALSE(afterBurritoIter);
 }
 
+void TextBufferTests::TestOverwriteChars()
+{
+    til::size bufferSize{ 10, 3 };
+    UINT cursorSize = 12;
+    TextAttribute attr{ 0x7f };
+    TextBuffer buffer{ bufferSize, attr, cursorSize, false, _renderer };
+    auto& row = buffer.GetRowByOffset(0);
+
+// scientist emoji U+1F9D1 U+200D U+1F52C
+#define complex1 L"\U0001F9D1\U0000200D\U0001F52C"

```

Comment:
presumably you're using defines because you can concatenate them with `L"X" L"Y" L"Z"`?

Reply:
Yeah exactly. Otherwise you'd have to use format strings which are even more verbose.